diff -wpruN '--exclude=*.orig' a~/neofetch a/neofetch
--- a~/neofetch	1970-01-01 00:00:00
+++ a/neofetch	1970-01-01 00:00:00
@@ -1414,8 +1414,10 @@ get_uptime() {
         ;;
 
         Solaris)
-            s=$(kstat -p unix:0:system_misc:snaptime | awk '{print $2}')
-            s=${s/.*}
+            boot=$(kstat -p unix:0:system_misc:boot_time | awk '{print $2}')
+            now=$(date +%s)
+
+            s=$((now - boot))
         ;;
 
         AIX|IRIX)
